src/integration/common/ads.canonical.js

```diff
@@ -0,0 +1,38 @@
+export default () => {
```

Comment:
I realised I have never looked at these tests in 'common' before. Are they running on something where the ads are always enabled?

Reply:
These are new integration tests which I've added so that we don't need to do E2Es - it's called from https://github.com/bbc/simorgh/blob/WSTEAMA-648-ads-on-homepage/src/integration/pages/homePage/ampTests.js#L15-L17 or https://github.com/bbc/simorgh/blob/WSTEAMA-648-ads-on-homepage/src/integration/pages/homePage/canonicalTests.js#L13-L15

but only if displayAds is set to true (which it is for arabic https://github.com/bbc/simorgh/blob/WSTEAMA-648-ads-on-homepage/src/integration/pages/homePage/arabic/amp.test.js#L4 and https://github.com/bbc/simorgh/blob/WSTEAMA-648-ads-on-homepage/src/integration/pages/homePage/arabic/canonical.test.js#L4)

There is then some code within the integration test setup to set the correct headers on the local environment so that the ads component will appear - https://github.com/bbc/simorgh/blob/WSTEAMA-648-ads-on-homepage/src/integration/integrationTestEnvironment.js#L36-L40)

Happy to explain how it works if you like.